src/binary/ContractionOptimizer.h

```diff
@@ -158,7 +164,9 @@ class einsum_ir::binary::ContractionOptimizer {
                int64_t                        i_target_n,
                int64_t                        i_target_k,
                bool                           i_br_gemm_support,
-               bool                           i_packed_gemm_support  );    
+               bool                           i_packed_gemm_support,
+               int64_t                        i_num_bytes_scalar_out = 4,
+               int64_t                        i_l2_cache_size = 1024 * 1024 );    
```

Comment:
Let\s remove the defaults to avoid silence performance issues deep down in the code.